src/explore-education-statistics-common/src/modules/charts/util/createDataSetCategories.ts

```diff
@@ -330,10 +338,11 @@ export default function createDataSetCategories(
     const value = Number(rawValue);
 
     if (includeNonNumericData && rawValue) {
-      acc.push([childDataSet, Number.isNaN(value) ? null : value] as Pair<
-        ChildDataSet,
-        number
-      >);
+      const nonNumericValue = returnSymbols ? rawValue : null;
+      acc.push([
+        childDataSet,
+        Number.isNaN(value) ? nonNumericValue : value,
```

Comment:
Would potentially suggest refactoring this a little to combine it with the second `if` block i.e.

```ts
if (!Number.isNaN(value)) {
  acc.push([childDataSet, value]);
} else if (includeNonNumericData && rawValue) {
  acc.push([childDataSet, returnNonNumericData ? rawValue : null]);
}
```

Actually, I'm now confused why we even need the `returnNonNumericData` / `returnSymbols` flag?

Couldn't this just be:

```ts
if (!Number.isNaN(value)) {
  acc.push([childDataSet, value]);
} else if (includeNonNumericData && rawValue) {
  acc.push([childDataSet, rawValue]);
}
```

Reply:
It was to not change the current behaviour for charts (the data sets are shown with null values), but I've checked with Cam and he says it's preferable to show the raw value for charts as well so I've got rid of  `returnSymbols` now.